src/tests/calculate.test.js

```diff
@@ -0,0 +1,16 @@
+import calculate from '../logic/calculate';
+
+describe('calculate', () => {
+  it('should return the sum of two numbers', () => {
+    const result = calculate({
+      total: 0,
+      next: null,
+      operation: null,
+    }, 'AC');
+    expect(result).toEqual({
+      total: null,
+      next: null,
+      operation: null,
+    });
+  });
```

Comment:
- [ ] Kindly add more tests covering most scenarios present in the calculate.js file.